Dockerfile-hpu

```diff
@@ -54,7 +54,7 @@ COPY proto proto
 
 RUN cargo build --release --bin text-embeddings-router -F grpc -F python --no-default-features && sccache -s
 
-FROM vault.habana.ai/gaudi-docker/1.15.0/ubuntu22.04/habanalabs/pytorch-installer-2.2.0:latest as base
+FROM vault.habana.ai/gaudi-docker/1.16.0/ubuntu22.04/habanalabs/pytorch-installer-2.2.2:latest as base
 
```

Comment:
Looks like https://vault.habana.ai/ui/native/gaudi-docker/1.16.2 is out.

Or maybe officially only 1.16.1: https://docs.habana.ai/en/v1.16.1/

Reply:
Have change it to 1.16.1.